AK/Error.h

```diff
@@ -11,7 +11,7 @@
 #include <AK/Variant.h>
 
 #if defined(AK_OS_SERENITY) && defined(KERNEL)
-#    include <LibC/errno_codes.h>
+#    include <errno_codes.h>
```

Comment:
:thinking:  why does this work? this shouldn't work.

Reply:
It's in Kernel context, and the Kernel already includes LibC files directly:
```
$ grep -Pirn 'include <(alloca|assert|byteswap|complex|ctype|dirent|dlfcn|elf|endian|errno_codes|errno|fcntl|fd_set|fenv|float|fnmatch|getopt|glob|grp|iconv|ifaddrs|inttypes|langinfo|libgen|limits|link|locale|mallocdefs|math|memory|mntent|netdb|nl_types|paths|poll|pthread|pty|pwd|regex|resolv|sched|search|semaphore|serenity|setjmp|shadow|signal|signal_numbers|spawn|stdarg|stddef|stdint|stdio_ext|stdio|stdlib|string|strings|sysexits|syslog|termcap|termios|time|ucontext|ulimit|unistd|utime|utmp|wchar|wctype)\.h>' Kernel/
Kernel/API/Graphics.h:12:#include <fcntl.h>
Kernel/API/Graphics.h:13:#include <stddef.h>
Kernel/API/Graphics.h:14:#include <stdio.h>
Kernel/API/Graphics.h:19:#include <unistd.h>
Kernel/API/TimePage.h:14:#    include <time.h>
Kernel/API/InodeWatcherEvent.h:15:#    include <limits.h>
Kernel/KBufferBuilder.h:12:#include <stdarg.h>
```
(And probably more.)

Like you said, this seems quite involved, and should be probably left as a FIXME. :/

Reply:
Most of the above examples *could* also come from Kernel/API/POSIX/, yes, but at least limits.h and stdarg.h don't exist there.

Reply:
Aight fair enough. Since it lets us keep AK off the naughty list, I'm good with this for now. If you don't already have something in the pipe i'll probably get to moving these declarations out of LibC and into Kernel so that nobody is allowed to include LibC/ prefixed headers (esp not from kernel code)

Reply:
Nope, I have nothing like that in the pipe, sorry :/